src/pointcloud/encode_pcd.py

```diff
@@ -0,0 +1,94 @@
+from .point_cloud import PointCloud
+from typing import List
+
+class Encoder():
```

Comment:
note: probably won't be necessary, but binary encoding is also a potential speed-up if that becomes necessary

Reply:
PCD can even be 'binary_compressed' !